mender-convert

```diff
@@ -568,6 +563,34 @@ do_mender_disk_image_to_artifact() {
     return 1
   fi
 
+  if [ ! -f ${mender_rootfs_image} ]; then
```

Comment:
Please work on root filesystem extracted from Mender Image, not a root filesystem taken from original raw disk image (i.e. Raspian or Bone Debian) where Mender related files are not installed.

Reply:
Will drop this change as, as it needs more re-work to actually work as intended.